serverless/develop/serverless-kafka-developer.adoc

```diff
@@ -43,10 +46,15 @@ include::modules/serverless-kafka-source-yaml.adoc[leveloffset=+2]
 [id="serverless-kafka-developer-broker"]
 == Kafka broker
 
+If a cluster administrator has configured your {ServerlessProductName} deployment to use Kafka broker as the default broker type, xref:../../serverless/develop/serverless-using-brokers.adoc#serverless-using-brokers-creating-brokers[creating a broker by using the default settings] creates a Kafka-based `Broker` object. If your {ServerlessProductName} deployment is not configured to use Kafka broker as the default broker type, you can still use the following procedure to create a Kafka-based broker.
+
 :FeatureName: Kafka broker
 include::snippets/technology-preview.adoc[leveloffset=+2]
 
-If a cluster administrator has configured your {ServerlessProductName} deployment to use Kafka broker as the default broker type, xref:../../serverless/develop/serverless-using-brokers.adoc#serverless-using-brokers-creating-brokers[creating a broker by using the default settings] creates a Kafka-based `Broker` object. If your {ServerlessProductName} deployment is not configured to use Kafka broker as the default broker type, you can still use the following procedure to create a Kafka-based broker.
+[IMPORTANT]
+====
+The Kafka broker Technology Preview is not supported on FIPS.
```

Comment:
Same here:
The Kafka broker, a Technology Preview feature, is currently not supported on FIPS.
or
The Kafka broker, which is currently in Technology Preview, is not supported on FIPS.